interceptor/go.mod

```diff
@@ -2,20 +2,4 @@ module globe-and-citizen/layer8/interceptor
 
 go 1.21.1
 
-require (
-	github.com/globe-and-citizen/layer8-utils v0.0.0-20230927121854-b5403af080d8
-	github.com/google/uuid v1.5.0
-)
-
-require (
-	github.com/aead/ecdh v0.2.0 // indirect
-	github.com/andybalholm/brotli v1.0.5 // indirect
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
-	github.com/goombaio/namegenerator v0.0.0-20181006234301-989e774b106e // indirect
-	github.com/klauspost/compress v1.16.3 // indirect
-	github.com/rogpeppe/go-internal v1.12.0 // indirect
-	github.com/valyala/bytebufferpool v1.0.0 // indirect
-	github.com/valyala/fasthttp v1.49.0 // indirect
-	golang.org/x/crypto v0.9.0 // indirect
-	gopkg.in/validator.v2 v2.0.1 // indirect
-)
+require github.com/google/uuid v1.5.0
```

Comment:
I don't understand here:  does it not use "github.com/globe-and-citizen/layer8-utils"?

Reply:
line 14 of `interceptor.go` has `github.com/globe-and-citizen/layer8-utils`. Should this not be reflected in the go.mod?

Reply:
That is confusing here, we already moved the utils to separate repo, and I changed the whole code, and if you go to `development` branch, you will see that there.

Oh I see the issue now, since I opened both the PR at the same time, so the code of development was old, and in this PR, as you can see, it's the old code that is why you are not seeing the utils from other Repo here, let me fix this.